NitroxServer/Resources/PrefabPlaceholdersGroupAsset.cs

```diff
@@ -1,19 +1,14 @@
-﻿using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Unity;
 
 namespace NitroxServer.Resources;
 
-public class PrefabPlaceholdersGroupAsset
+public record struct PrefabPlaceholdersGroupAsset(string ClassId, IPrefabAsset[] PrefabAssets) : IPrefabAsset
```

Comment:
We discussed in Discord that it wasn't too heavy in here so it was fine to still use struct because remaking it into a class would break the system around. Resolving the conversation